scripts/import_macos_pkg_resources.py

```diff
@@ -0,0 +1,127 @@
+#! /usr/bin/env python3
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+# This script must be executed at the root of the repository.
+
+import os
+import xml.etree.ElementTree as ET
+
+# List of translation keys
+keys = [ "macosinstaller.title",
+         "macosinstaller.unsupported_version.title",
+         "macosinstaller.unsupported_version.message",
+         "macosinstaller.previous_build.title",
+         "macosinstaller.previous_build.message",
+         "macosinstaller.welcome.message1",
+         "macosinstaller.welcome.message2",
+         "macosinstaller.conclusion.title",
+         "macosinstaller.conclusion.message1",
+         "macosinstaller.conclusion.message2",
+         "macosinstaller.conclusion.message3" ];
+
+# Simple method that retrieves the text value of a localized string.
+def getTranslation(root, string, nodeName):
+    for node in root.findall(f"./context/message[@id='{string}']"):
+        for child in node:
+            if child.tag != nodeName:
+                continue
+            if child.text == None:
+                return ''
+            return child.text
+    return ''
+
+# Generate the translation folder for a particular locale. `nodeTag` can be
+# `source` (for english only) or `translation` (for anything else)
+def translate(root, locale, nodeTag):
```

Comment:
You could avoid having `nodeTag` at all in the function, and just have at the beginning

```
noteTag = "source" if locale == "en" else "translation"
```

Right now the logic is spread a bit all over the place.